### PR TITLE
15 driver profiles

### PR DESCRIPTION
**Pull request by Something-Inconspicuous**
Added data class driver profiles; more parameters may be added in the future

### CODE REVIEW

**Comment by Something-Inconspicuous** [comment]
FIXME: There may be a potential bug when the power scale is even where the joystick axis result will always be positive.
This might be fixed by using `sign(x) * |x^(a)|` instead of `x^(a)`
For example, `(-0.9)^(2.0) > 0`, but `sign(-0.9) * |(-0.9)^(2.0)| < 0`

**Comment by jstN0body** [comment]
@Something-Inconspicuous please take a look at the following [pull request](https://github.com/Team2537/2024Crescendo/pull/28)

File: src/main/java/frc/robot/util/DefaultDriverProfile.kt
Comment (jstN0body) [inline src/main/java/frc/robot/util/DefaultDriverProfile.kt:1]: Make sure to remove this in future commits

File: src/main/java/lib/controllers/ProfileController.kt
Comment (jstN0body) [inline src/main/java/lib/controllers/ProfileController.kt:1]: Open to suggestions on how to simplify this, changes should be made before merging to dev

**Review by jstN0body — COMMENTED** [review]
This all looks good to me, @falOn-Dev please double check before we merge.

**Pull request merged**